src/clarity/buttons/_buttons.clarity.scss

```diff
@@ -265,13 +264,23 @@ $clr-button-appearance-map: (
         }
 
         &.btn-sm {
-            @include clr-getStylesFromMap($clr-button-appearance-map,standard);
+            @include clr-getStylesFromMap($clr-button-appearance-map, standard);
+
+            clr-icon {
```

Comment:
Whats the size of the icons in regular buttons? Am assuming its the default size: 16px?